libs/deso-protocol-types/src/lib/deso-types-custom.ts

```diff
@@ -344,3 +344,11 @@ export interface MetaMaskInitResponse {
 export interface RequestOptions {
   broadcast?: boolean;
 }
+
```

Comment:
```suggestion
export enum MessagingGroupOperation {
  DEFAULT_KEY = 'DefaultKey',
  CREATE_GROUP = 'CreateGroup',
  ADD_MEMBERS = 'AddMembers',
}

```

we should probably add this enum to control the `MessagingGroupOperation` type